src/app/base/components/AppSideNavigation/AppSideNavCollapseToggle/AppSideNavCollapseToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, Icon, Tooltip } from "@canonical/react-components";
+import classNames from "classnames";
+
+const AppSideNavCollapseToggle = ({
+  isCollapsed,
+  setIsCollapsed,
+  className,
+}: {
+  isCollapsed: boolean;
+  setIsCollapsed: (isCollapsed: boolean) => void;
+  className?: string;
+}): JSX.Element => {
+  return (
+    <Tooltip
+      message={<>{!isCollapsed ? "collapse" : "expand"} ( [ )</>}
```

Comment:
Would it be better to wrap the [ symbol in an inline code block like this? `[`

Just for greater clarity